politeiad/api/v1/v1.go

```diff
@@ -21,13 +21,15 @@ type RecordStatusT int
 
 const (
 	// Routes
-	IdentityRoute             = "/v1/identity/"       // Retrieve identity
-	NewRecordRoute            = "/v1/newrecord/"      // New record
-	UpdateUnvettedRoute       = "/v1/updateunvetted/" // Update unvetted record
-	UpdateVettedRoute         = "/v1/updatevetted/"   // Update vetted record
-	UpdateVettedMetadataRoute = "/v1/updatevettedmd/" // Update vetted metadata
-	GetUnvettedRoute          = "/v1/getunvetted/"    // Retrieve unvetted record
-	GetVettedRoute            = "/v1/getvetted/"      // Retrieve vetted record
+	IdentityRoute               = "/v1/identity/"          // Retrieve identity
+	NewRecordRoute              = "/v1/newrecord/"         // New record
+	UpdateUnvettedRoute         = "/v1/updateunvetted/"    // Update unvetted record
+	UpdateVettedRoute           = "/v1/updatevetted/"      // Update vetted record
+	UpdateVettedMetadataRoute   = "/v1/updatevettedmd/"    // Update vetted metadata
+	UpdateUnvettedMetadataRoute = "/v1/updateunvettedmd/"  // Update unvetted metadata
+	GetUnvettedRoute            = "/v1/getunvetted/"       // Retrieve unvetted record
+	GetVettedRoute              = "/v1/getvetted/"         // Retrieve vetted record
+	InventoryByStatusRoute      = "/v1/inventorybystatus/" // Inventory record tokens by status
```

Comment:
not sure if this is the best naming for this route

Reply:
I think that's fine.  I'd go with just `Inventory` if it wasn't already taken, but since its already taken the more verbose `InventoryByStatus` is fine.

Reply:
Agreed mate :+1: